filament/src/details/RenderTarget.cpp

```diff
@@ -34,7 +34,7 @@ struct RenderTarget::BuilderDetails {
     uint32_t mWidth{};
     uint32_t mHeight{};
     uint8_t mSamples = 1;   // currently not settable in the public facing API
-    uint8_t mLayerCount = 0;// currently not settable in the public facing API
+    uint8_t mLayerCount = 0;
```

Comment:
Though I can change it to 1 since layer==1 still means a regular RT, I'm not sure what you meant. Could you elaborate on where it could cause an issue? I tested it with both Vulkan and OpenGL backends.

Reply:
Sorry not about barriers, but when the image is created.

Here's the validation error it tripped:
https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-VkImageViewCreateInfo-imageViewType-04973

Reply:
I see. Makes sense. Let me update it.